termui/bug_table.go

```diff
@@ -340,8 +340,13 @@ func (bt *bugTable) renderFooter(v *gocui.View, maxX int) {
 
 func (bt *bugTable) cursorDown(g *gocui.Gui, v *gocui.View) error {
 	_, y := v.Cursor()
-	y = minInt(y+1, bt.getTableLength()-1)
 
+	// If we are at the bottom of the page, switch to the next one.
+	if y+1 > bt.getTableLength()-1 {
```

Comment:
I see where it came from, but that's a weird test ;)
```suggestion
 	if y+1 >= bt.getTableLength() {
```
?

Reply:
Yep, this might be more readable :)